ansible/install_keystone.yaml

```diff
@@ -55,3 +55,16 @@
     environment:
       PATH: "{{ oc_env_path }}"
       KUBECONFIG: "{{ kubeconfig }}"
+
+  # This workaround can be removed when
+  # https://github.com/openstack-k8s-operators/keystone-operator/issues/30 is
```

Comment:
Can you change this comment to this?

https://issues.redhat.com/browse/OSPK8-247

Reply:
That's a different issue. I can file a new one, though.